Source/ACE.Server/WorldObjects/Player_Inventory.cs

```diff
@@ -480,11 +480,38 @@ public void HandleActionPutItemInContainer(ObjectGuid itemGuid, ObjectGuid conta
             }).EnqueueChain();
         }
 
+        private bool IsAttuned(ObjectGuid itemGuid, out bool isAttuned)
+        {
+            WorldObject item = GetInventoryItem(itemGuid);
+            if (item == null)
+            {
+                isAttuned = false;
+                return false;
```

Comment:
Should it be the responsibility for the main return value of IsAttuned to be if the item is inventory?

It's cool that IsAttuned checks for this, but if the calling function needs to differentiate between an item not being in inventory / unattuned items, maybe there should be a separate check in the calling function?